Sources/Deprecations+Removals.swift

```diff
@@ -235,21 +238,6 @@ extension Action {
 }
 
 // Renamed Enum cases
-
-extension Event {
```

Comment:
These are no longer necessary, since SourceKit has auto-suggestion that covers capitalisation.